### DISCUSSION TITLE
How can I use zeroui with a zerotier controller outside docker

### DISCUSSION THREAD

**Discussion by qiuwei** [dec0dOS/zero-ui#23] [Q&A]
I am trying only running zeroui in docker and the zerotier controller on the localhost.
I have made the zerotier controller accept management from all IPs.

The local.conf for zerotier looks like this:

```
{"settings": {"allowManagementFrom": ["0.0.0.0/0"]}}
```

I can manage the controller from a remote server.
However,  zeroui in the docker container seems not to be able to send requests to the zerotier controller.

**Comment by dec0dOS** [comment]
Hello, @qiuwei! You could try adding `network_mode: host` to your ZeroUI service in `docker-compose.yml` file.
